NuKeeper/Configuration/CommandLineParser.cs

```diff
@@ -35,8 +35,18 @@ private static RepositoryModeSettings ReadSettingsForRepositoryMode(CommandLineA
             var githubToken = settings.GithubToken;
             var githubRepoUri = settings.GithubRepositoryUri;
 
+            Uri githubApiBase;
+            if (settings.GithubApiEndpoint != null)
+            {
+                githubApiBase = settings.GithubApiEndpoint;
```

Comment:
In your e.g. this isn't needed for github.com as code to generate the api url was written for github.com. It doesn't work on an internal github with a different api url structure.

I agree that we want to standardise args (issue #21) and use something like the organisation mode version. How far away is this? If it's near than I will withdraw this. Otherwise, it allows me to test other things today in the meantime.